src/components/Video.js

```diff
@@ -1,26 +1,53 @@
-import React from "react";
+import React, { useState } from "react";
 import "../../node_modules/video-react/dist/video-react.css";
-import { Player } from "video-react";
+// import { Player, BigPlayButton } from "video-react";
+import style from "styled-components";
+import ReactPlayer from "react-player/lazy";
 
-const videoContainer = {
-  maxWidth: "50%",
-  width: "-webkit-fill-available",
-  margin: "0 auto",
-};
+const VideoContainer = style.div`
+  width: 100%;
+  position: relative;
+`;
+
+const VideoPlayPause = style.img`
+  top: calc(50% - 35px);
+  left: calc(50% - 35px);
+  position: absolute;
+  display: ${(props) => (props.playing ? "none" : "flex")};
+`;
 
 const Video = (props) => {
-  console.log("props.video", props.video);
+  const [playerState, setPlayerState] = useState({
+    playing: false,
+    controls: false,
+  });
+
+  const handlePlayPause = () => {
+    setPlayerState({ ...playerState, playing: !playerState.playing });
+  };
+
   return (
-    <div style={videoContainer}>
-      <Player
-        playsInline
-        src={
-          props.video
-            ? props.video.location
-            : `${process.env.REACT_APP_S3VIDEOS}billieeilish--you_should_see_me_in_a_crown-king_me_-coLerbRvgsQ.mp4`
-        }
-      />
-    </div>
+    <>
+      <VideoContainer onClick={handlePlayPause}>
+        <ReactPlayer
+          playsinline
+          fileConfig={{ attributes: { poster: props.video.thumbnail } }}
+          playing={playerState.playing}
+          controls={playerState.controls}
+          width="100%"
+          height="100%"
+          url={
+            props.video
+              ? props.video.location
+              : `${process.env.REACT_APP_S3VIDEOS}billieeilish--you_should_see_me_in_a_crown-king_me_-coLerbRvgsQ.mp4`
+          }
+        ></ReactPlayer>
+        <VideoPlayPause
+          playing={playerState.playing}
+          src="../images/icon-video-play.svg"
+        />
+      </VideoContainer>
+    </>

```

Comment:
This new player looks like it is going to give us a lot more control of styling.